src/field_theory/polynomial_galois_group.lean

```diff
@@ -208,6 +208,62 @@ begin
   { rwa [ne.def, mul_eq_zero, map_eq_zero, map_eq_zero, ←mul_eq_zero] }
 end
 
+lemma mul_splits_in_splitting_field_of_mul {p₁ q₁ p₂ q₂ : polynomial F}
+  (hq₁ : q₁ ≠ 0) (hq₂ : q₂ ≠ 0) (h₁ : p₁.splits (algebra_map F q₁.splitting_field))
+  (h₂ : p₂.splits (algebra_map F q₂.splitting_field)) :
+  (p₁ * p₂).splits (algebra_map F (q₁ * q₂).splitting_field) :=
+begin
+  apply splits_mul,
+  { rw ← (splitting_field.lift q₁ (splits_of_splits_of_dvd _
+      (mul_ne_zero hq₁ hq₂) (splitting_field.splits _) (dvd_mul_right q₁ q₂))).comp_algebra_map,
```

Comment:
Arguably we could extract this lemma that `(algebra_map F (q₁ * q₂).splitting_field) = (↑(splitting_field.lift q₁ _).comp (algebra_map F q₁.splitting_field))`, but I don't think that will improve the readability too much.

An alternative would be to have instances for `algebra q₁.splitting_field (q₁ * q₂).splitting_field` and `scalar_tower F q₁.splitting_field (q₁ * q₂).splitting_field`, then rewrite with `scalar_tower.algebra_map_eq`. Also doesn't sound too convincing to me.

So in conclusion, I'm okay with this as-is but if you want to improve it, this is where I'd start.